Surface_mesher/include/CGAL/Surface_mesher/Surface_mesher.h

```diff
@@ -298,6 +298,7 @@ namespace CGAL {
 
           for(bool exit = false; ; exit = true)
           {
+            CGAL_USE(exit);
```

Comment:
That loop `for(bool exit = false; ; exit = true)` was a bad trick to execute the body exactly twice. But the `return false` line 324 is no longer conditional, so the loop will be executed only once. I agree with the patch, because I do not want to do refactoring in Surface_mesher.